src/ui/control/geolocate_control.js

```diff
@@ -267,7 +267,7 @@ class GeolocateControl extends Evented {
             this._accuracyCircleMarker.setLngLat(center).addTo(this._map);
             this._userLocationDotMarker.setLngLat(center).addTo(this._map);
             this._accuracy = position.coords.accuracy;
-            if (this.options.trackUserLocation && this.options.showAccuracyCircle) {
+            if (this.options.showUserLocation && this.options.showAccuracyCircle) {
```

Comment:
One final point now is just to update the API doc to note that `showAccuracyCircle` only has effect when `showUserLocation` is set to true.

Reply:
👍 5c3713335508f31b3273731ab29a32cbc5354fdc